examples/bri-3/src/bri/transactions/api/transactions.controller.ts

```diff
@@ -47,7 +53,7 @@ export class TransactionController {
   async updateTransaction(
     @Param('id') id: string,
     @Body() requestDto: UpdateTransactionDto,
-  ): Promise<void> {
+  ): Promise<UpdateTransactionDto> {
```

Comment:
same comment as above, lets not change api response using request dto in mapper PR
imo we should define what our apis return and do it as separate issue